### PR TITLE
Updated JAX event with new links, talks and more

### PR DESCRIPTION
**Pull request by merveenoyan**

### CODE REVIEW

File: jax-controlnet-sprint/README.md [hunk 1]
```diff
@@ -49,12 +51,20 @@ For issues with Flax/JAX, Diffusers, Datasets or for questions that are specific
...
 
-> 💡 Note: If you have signed up on the google form, but you are not in the Discord channel, please leave a message on [(TODO) the official forum announcement] () and ping `@mervenoyan`, `@sayakpaul`, and `@patrickvonplaten`. We might take a day to process these requests.
+> 💡 Note: After 10th of April, if you have signed up on the google form, but you are not in the Discord channel, please leave a message on [the official forum announcement] (https://discuss.huggingface.co/t/controlling-stable-diffusion-with-jax-and-diffusers-using-v4-tpus/35187/2) and ping `@mervenoyan`, `@sayakpaul`, and `@patrickvonplaten`. We might take a day to process these requests.
+
+## Talks
+
+We will have talks from folks working at JAX & TPU teams at Google, diffusers and ethics teams at Hugging Face and awesome open-sorcerers working in generative AI. We will update this post with links to the talks, so keep an eye here or on Discord in diffusion models core-announcements channel and set your reminders!
+
+- [Link to the talks on 17th of April](https://www.youtube.com/watch?v=SOj2sxgvFe0), we will be hosting Andreas Steiner, Margaret Mitchell and Boris Dayma.
```
Comment (osanseviero) [inline jax-controlnet-sprint/README.md:68]: Maybe let's do a table similar to https://github.com/huggingface/transformers/tree/main/examples/research_projects/jax-projects#talks  (ok if all links go to same video) so we also have the topics

**Review by osanseviero — COMMENTED** [review]
Nice!

**Comment by patrickvonplaten** [comment]
Cool !

File: jax-controlnet-sprint/README.md (hunk 1 above)
Comment (merveenoyan) [reply jax-controlnet-sprint/README.md:68]: Looks neat!

**Pull request merged**